spectator-ext-jvm/src/test/java17/com/netflix/spectator/jvm/JavaFlightRecorderTest.java

```diff
@@ -0,0 +1,35 @@
+package com.netflix.spectator.jvm;
+
+import com.netflix.spectator.api.DefaultRegistry;
+import com.netflix.spectator.api.Id;
+import com.netflix.spectator.api.Measurement;
+import com.netflix.spectator.api.Registry;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class JavaFlightRecorderTest {
+
+  @Test
+  public void test() throws Exception {
+    Registry registry = new DefaultRegistry();
+    ExecutorService executor = Executors.newSingleThreadExecutor();
+    try (var closable = JavaFlightRecorder.monitorDefaultEvents(registry, executor)) {
+      Thread.sleep(2000);
+    }
+    executor.shutdownNow();
+
+    Assertions.assertEquals(3, registry.measurements().count());
+    List<String> distinctMeasures = registry.measurements()
+      .map(Measurement::id)
+      .map(Id::name)
+      .distinct()
+      .toList();
+    Assertions.assertEquals(1, distinctMeasures.size());
```

Comment:
How do you know a GC event or something else couldn't have happened in the 2s window? Just want to make sure the test will not be flakey.

Reply:
We don't, but if you're concerned about the `cpuLoad` metric, I'll find a different metric to test off and not check the total count, but that we see the expected event(s).

Reply:
Just checking for some events seems reasonable if we cannot guarantee the number. So long as the test is robust and doesn't spuriously fail.

Reply:
I've rewritten that part of the test.